benchmarks/linear-elastic-plate-with-hole/workflow_creation_guide.md

```diff
@@ -0,0 +1,930 @@
+    input_summary_parameter_file = tool1_parameter_files.concat(tool2_parameter_files)
+    input_summary_mesh = tool1_meshes.concat(tool2_meshes)
+    input_summary_solution_field = tool1_solution_fields.concat(tool2_solution_fields)
+    input_summary_metrics = tool1_summary_metrics.concat(tool2_summary_metrics)
+    
+    // Generate summaries
+    def ch_benchmark = Channel.value(params.benchmark)
+    def ch_summarise_python_script = Channel.value(file('summarise_results.py'))
+    summary(ch_summarise_python_script,
+            input_summary_configuration,
+            input_summary_parameter_file,
+            input_summary_mesh,
+            input_summary_metrics,
+            input_summary_solution_field,
+            ch_benchmark,
+            ch_tools)
+}
+```
+
+#### Nextflow Configuration (nextflow.config)
+
+```nextflow-config
+conda {
+   enabled = true
+}
+
+params.result_dir = "nextflow_results/${params.benchmark}"
+
+prov {
+   formats {
+      dag {
+         file = "${params.result_dir}/nf_prov_dag.html"
+         overwrite = true
+      }
+      legacy {
+         file = "${params.result_dir}/nf_prov_legacy.json"
+         overwrite = true
+      }
+      wrroc {
+         agent {
+            name = 'Your Name'
+            orcid = 'https://orcid.org/0000-0000-0000-0000'
+         }
+         file = "${params.result_dir}/ro-crate-metadata.json"
+         license = 'https://spdx.org/licenses/MIT'
+         overwrite = true
+      }
+   }
+}
+```
+
+#### Tool-Specific Nextflow Workflow (tool1/tool1.nf)
+
+```nextflow
+params.tool = "tool1"
+
+process run_simulation {
+    publishDir "${params.result_dir}/${params.tool}/"
+    conda './tool1/environment_simulation.yml' 
+
+    input:
+    path python_script
+    tuple val(configuration), path(parameter_file), path(mesh_file)
+
+    output:
+    tuple val(configuration), path("solution_field_data_${configuration}.zip"), path("solution_metrics_${configuration}.json")
+
+    script:
+    """
+    python3 $python_script \
+        --input_parameter_file $parameter_file \
+        --input_mesh_file $mesh_file \
+        --output_solution_file_zip "solution_field_data_${configuration}.zip" \
+        --output_metrics_file "solution_metrics_${configuration}.json"
+    """
+}
+
+workflow tool1_workflow {
+    take: 
+    mesh_data // tuple(configuration, parameters, mesh) 
+    result_dir
+
+    main:
+    params.result_dir = result_dir
+    run_sim_script = Channel.value(file('tool1/run_tool1_simulation.py'))
+    output_process_run_simulation = run_simulation(run_sim_script, mesh_data)
+
+    emit:
+    output_process_run_simulation
+}
+```
+
+### Step 6: Implement Simulation Scripts
+
+Each tool needs a standardized simulation script:
+
+```python
+import json
+import numpy as np
+from argparse import ArgumentParser
+from pathlib import Path
+from pint import UnitRegistry
+
+def run_simulation(parameter_file: str, mesh_file: str, 
```

Comment:
the important steps (unit conversion, metrics and zip output) should be discussed, then link the file